tools/dds/dds-sniffer/rs-dds-sniffer.cpp

```diff
@@ -272,72 +386,79 @@ uint32_t dds_sniffer::calc_max_indentation() const
+                                           const char * topic_name,
+                                           bool discovered ) const
 {
-    if (_print_machine_readable)
+    if( _print_machine_readable )
     {
-        std::cout << "DataWriter," << guid << "," << topic_name << (discovered ? ",discovered" : ",removed") << std::endl;
+        std::cout << "DataWriter," << guid << "," << topic_name << ( discovered ? ",discovered" : ",removed" ) << std::endl;
     }
     else
     {
-        std::cout << "DataWriter  " << guid << " publishing topic '" << topic_name 
-                  << (discovered ? "' discovered" : "' removed") << std::endl;
+        std::cout << "DataWriter  " << guid << " publishing topic '" << topic_name
+                  << ( discovered ? "' discovered" : "' removed" ) << std::endl;
     }
 }
 
-void dds_sniffer::print_reader_discovered( librealsense::dds::dds_guid guid, const char* topic_name, bool discovered ) const
+void dds_sniffer::print_reader_discovered( librealsense::dds::dds_guid guid,
+                                           const char * topic_name,
+                                           bool discovered ) const
 {
-    if (_print_machine_readable)
+    if( _print_machine_readable )
     {
-        std::cout << "DataReader," << guid << "," << topic_name << (discovered ? ",discovered" : ",removed") << std::endl;
+        std::cout << "DataReader," << guid << "," << topic_name << ( discovered ? ",discovered" : ",removed" ) << std::endl;
     }
     else
     {
         std::cout << "DataReader  " << guid << " reading topic '" << topic_name
-                  << (discovered ? "' discovered" : "' removed") << std::endl;
+                  << ( discovered ? "' discovered" : "' removed" ) << std::endl;
     }
 }
 
-void dds_sniffer::print_participant_discovered( librealsense::dds::dds_guid guid, const char* participant_name, bool discovered ) const
+void dds_sniffer::print_participant_discovered( librealsense::dds::dds_guid guid,
+                                                const char * participant_name,
+                                                bool discovered ) const
 {
-    if (_print_machine_readable)
+    if( _print_machine_readable )
     {
         std::cout << "Participant," << guid << "," << participant_name
-                  << (discovered ? ",discovered" : ",removed") << std::endl;
+                  << ( discovered ? ",discovered" : ",removed" ) << std::endl;
     }
     else
     {
         uint16_t tmp( 0 );
         memcpy( &tmp, &guid.guidPrefix.value[GUID_PROCESS_LOCATION], sizeof( tmp ) );
-        std::cout << "Participant " << guid << " " << participant_name << "_" << std::hex << tmp
-                  << std::dec << (discovered ? " discovered" : " removed") << std::endl;
+        std::cout << "Participant " << guid << " " << participant_name << "_" << std::hex << tmp << std::dec
+                  << ( discovered ? " discovered" : " removed" ) << std::endl;
```

Comment:
No need to reset to `std::dec` at the end ? 

This setting remain valid for next print IMO.

```
std::cout << std::hex << int( 20 ) << std::endl;
std::cout << int( 20 ) << std::endl;
```

![image](https://user-images.githubusercontent.com/64067618/180710995-667bc279-8097-44bf-b1fa-a92a72f5c08c.png)

Reply:
I reset to `std::dec` after printing the hex GUID (clang formatter just moved it to previous line)

Reply:
Oh I see it now, my bad 🥇